src/ui.dropdown-list/UDropdownList.vue

```diff
@@ -71,7 +71,7 @@
         @slot Use it to add something instead of empty state.
         @binding {string} emptyStyles
       -->
-      <slot name="empty" :empty-styles="optionClasses">
+      <slot name="empty" :empty-styles="optionAttrs">
```

Comment:
here some logic was used in USelect and now missing. Please check it with @Explicit12 and remove this slot property or fix it. optionAttrs - is not a right way because it is object, not a string.